README.md

```diff
@@ -14,7 +14,7 @@ This library implements the Ably REST and Realtime client APIs.
 
 ### REST API
 
-In respect of the Ably REST API, this library targets the Ably 1.1 client library specification,
+In respect of the Ably REST API, this library targets the Ably 1.2 client library specification,
```

Comment:
> Should that also say 1.2?

True. Fixed at e9b274a.



> Also, why does the library need to be upgraded incrementally? Are we saying users have to upgrade to 1.1, then to 1.2?

As I understand it, this means that version 1.1 (now, 1.2) doesn't implement the whole 1.1 spec. While we haven't really decided this, I'd expect it will be the same case for 1.2. Version 1.2.0 can be released as soon as we have a partially spec-compliant version onto which 1.2 features can be added in a backwards-compatible way in further minor releases.

Reply:
> Version 1.2.0 can be released as soon as we have a partially spec-compliant version onto which 1.2 features can be added in a backwards-compatible way in further minor releases.

Ok, I agree with that, but I'm still confused by the sentence in the README, since it seems to suggest that the user needs to do some form of incremental upgrade (e.g. upgrade to 1.2.1, then 1.2.2, then 1.2.3 etc.), but I don't believe that to be the case.

Perhaps the section would be clearer as something like:

> This library targets the Ably 1.2 client library specification, although as of release 1.2.0 the following are not implemented and will be covered in future 1.2.x releases:
>
> [list of unimplemented things]
>
> If any of these unimplemented features are a high priority for your use-case then please contact Ably customer support. Pull Requests are also welcomed.

Reply:
See 1e4ff57.